web/src/components/__tests__/SearchFeedback.spec.tsx

```diff
@@ -50,4 +51,19 @@ describe('SearchFeedback', () => {
     )
     expect(getByText('feedback:send')).toBeTruthy()
   })
+
+  it('should not allow sending search feedback if query term is removed', async () => {
+    const { getByText, rerender } = renderWithTheme(
+      <SearchFeedback cityCode={cityCode} languageCode={languageCode} query='ab' noResults />,
+    )
+    expect(getByText('feedback:send')).toBeTruthy()
```

Comment:
Probably we should add something like
```suggestion
    expect(getByText('feedback:send')).toBeTruthy()
    expect(getByText('feedback:send')).toBeEnabled()
```

Reply:
Ooooops, you're absolutely right, thanks!